src/rules/pycodestyle/rules/mod.rs

```diff
@@ -0,0 +1,41 @@
+use rustpython_parser::ast::{Cmpop, Expr, ExprKind};
+
+use crate::ast::helpers::{create_expr, unparse_expr};
+use crate::source_code::Stylist;
+
+pub fn compare(left: &Expr, ops: &[Cmpop], comparators: &[Expr], stylist: &Stylist) -> String {
```

Comment:
This is small enough that I'd be fine inlining it at each call site. I think it used to be "longer".

Alternatively, you could put this in a `helpers.rs` file in `pycodestyle`. Totally up to you.

Reply:
OK, I put the function in each rule file that uses it (two of them)

Reply:
> This is small enough that I'd be fine inlining it at each call site.

Just a note that Rust supports `#[inline]` for inlining functions ... personally I'd prefer us to not duplicate code in general even if it is just 10 lines.

Reply:
I looked up #[inline] but I'm not sure I understand how to use it right away (very new to Rust). Perhaps I misunderstood the original request. I will look into it some more, or if you have an example of how to do it in this case I'd really appreciate it.

Reply:
`#[inline]` is a hint to the compiler that it should inline the function wherever it's used. It's typically not needed, since the compiler is pretty good at figuring out where that's a helpful behavior.

Reply:
My suggestion to inline the code was more of a WET vs. DRY thing :) But I don't feel strongly.

Reply:
No worries. The term "inlining" is somewhat ambiguous ... I don't think Charlie meant inlining as in avoiding a function call. `#[inline]` can be added  for performance when a function is called very often, see [Inlining](https://nnethercote.github.io/perf-book/inlining.html) ... but this is not something we probably actually want here ... I just wanted to mention it.^^